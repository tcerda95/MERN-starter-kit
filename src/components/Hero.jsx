import React from 'react';
import NavLinks from './NavLinks';
import 'bulma/css/bulma.css';

export default function Hero(props) {
  return (
    <section className="hero is-info">
      <div className="hero-body">
        <div className="container">
          <h1 className="title">
            Social Media Comments
          </h1>
          <h2 className="subtitle">
            MERN application example
          </h2>
          <NavLinks />
        </div>
      </div>
    </section>
  );
}
