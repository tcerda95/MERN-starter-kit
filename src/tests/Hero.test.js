import React from 'react';
import Hero from '../components/Hero';
import { shallow } from 'enzyme';

describe('React Hero testing', () => {
  let wrapper;

  beforeEach(() => { wrapper = shallow(<Hero />); });

  it('renders two <div>', () => {
    expect(wrapper.find('div')).toHaveLength(2);
  });

  it('renders one .title', () => {
    expect(wrapper.find('.title')).toHaveLength(1);
  });
});