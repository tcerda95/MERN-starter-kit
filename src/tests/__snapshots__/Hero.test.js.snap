// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Hero testing is better snapshot testing for regression render tests 1`] = `
<section
  className="hero is-info"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        Social Media Comments
      </h1>
      <h2
        className="subtitle"
      >
        MERN application example
      </h2>
    </div>
  </div>
</section>
`;
